Migrate cosign off the unmaintained chrismellard ACR credential helper.
Upstream: https://github.com/sigstore/cosign/pull/5099
Advisory: https://pkg.go.dev/vuln/GO-2026-6225

diff --git a/cmd/cosign/cli/options/registry.go b/cmd/cosign/cli/options/registry.go
index 495ae60c9..6ba539455 100644
--- a/cmd/cosign/cli/options/registry.go
+++ b/cmd/cosign/cli/options/registry.go
@@ -25,7 +25,7 @@ import (
 	"os"
 
 	ecr "github.com/awslabs/amazon-ecr-credential-helper/ecr-login"
-	"github.com/chrismellard/docker-credential-acr-env/pkg/credhelper"
+	"github.com/osscontainertools/docker-credential-acr/pkg/credhelper"
 	"github.com/google/go-containerregistry/pkg/authn"
 	"github.com/google/go-containerregistry/pkg/authn/github"
 	"github.com/google/go-containerregistry/pkg/name"
